vyper/parser/lll_node.py

```diff
@@ -213,7 +212,8 @@ def __init__(self,
                     self.gas = sum([arg.gas for arg in self.args]) + 30
                 if self.value == 'if_unchecked':
                     self.gas = self.args[0].gas + self.args[1].gas + 17
-        elif self.value is None and isinstance(self.typ, NullType):
+        elif self.value is None:
+            # TODO revisit this
```

Comment:
Needs a way better comment than this if you're going to leave a TODO

Reply:
thanks, resolved